### PR TITLE
Request

### PR DESCRIPTION
**Pull request by eksite**

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -9,11 +9,14 @@ const Head = styled.header`
...
+    </Link>
+  ));
 
-const Header = () => (
-  <Head>
-    <Link to="/">Home</Link>
-    <Link to="/fetch">fetchOnly</Link>
-  </Head>
-);
+  return res;
```
[the remark this answers is not in the document]
Comment (ilibaz) [reply src/components/header.js]: Or Fragment.
Anyway, set of Links is not a Header, should be a separate component like Navigation

File: src/components/Match.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 import MatchID from './MatchID';
+import { useParams } from 'react-router-dom';
 
-const Match = props => <MatchID data={props.match.params.number}></MatchID>;
+const Match = props => {
+    const { siteID } = useParams();
```
Comment (ilibaz) [inline src/components/Match.js]: maybe use `camelCase`

File: src/components/header.js [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-);
+// const Head = styled.header`
+//   display: flex;
+//   background-color: black;
+//   color: white;
+//   flex-direction: row;
+//   justify-content: c;
+// `;
+const Header = ({ items }) => {
+  const res = Object.entries(items).map(arr => (
```
Comment (ilibaz) [inline src/components/header.js]: no need for temp res variable

File: src/components/header.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ilibaz) [reply src/components/header.js]: This is still an issue

File: src/components/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+import { Link } from 'react-router-dom';
+
+const Links = ({ items }) =>
+    Object.entries(items).map(arr => (
+        <Link key={arr[1]} to={arr[1]}>
+            {arr[0]}
+        </Link>
+    ));
+export default Links;
```
[the remark this answers is not in the document]
Comment (osele001) [reply src/components/Navigation.js]: It is good that you split Header component and create a new Navigation component.
However, you still have the problem with missing outer html element. You should always have an outer html element in exported components.
**Hint**
Just add a fragment or <div> around your Object.entries call.
`const Links = ({ items }) => (`
`    <>`
`    // ... `
`    </>`
`)`

File: src/components/App.js [hunk 5]
```diff
@@ -1,20 +1,22 @@
 import React from 'react';
-import Header from './header';
+import Header from './Header';
 import Match from './Match';
 import { Route, Switch } from 'react-router-dom';
-import Fetch from './fetch';
+import Fetch from './Fetch';
 const links = { home: '/home', heroes: '/heroes', match: '/match' };
```
Comment (ilibaz) [inline src/components/App.js]: idea for refactoring (maybe in other PR)

```
const links = [
  {
    title: 'The page title',
    text: 'Home',
    url: '/home'
  },
  {
  ...
  }
];
```

**Pull request merged**